tests/acceptance/features/webUISharingPublic/shareByPublicLink.feature

```diff
@@ -101,13 +101,75 @@ Feature: Share by public link
     When the public uses the webUI to access the last public link created by user "user1"
     Then it should not be possible to delete file "lorem.txt" using the webUI
 
+  Scenario: creating a public link with "Editor" role makes it possible to upload a file
+    Given user "user1" has shared folder "simple-folder" with link with "read, change, create, delete" permissions
+    When the public uses the webUI to access the last public link created by user "user1"
+    And the user uploads file "new-lorem.txt" using the webUI
+    Then file "new-lorem.txt" should be listed on the webUI
+    And as "user1" file "simple-folder/new-lorem.txt" should exist
+
+  Scenario: creating a public link with "Editor" role makes it possible to upload a file inside a subdirectory
```

Comment:
This should have been the following, but, I just noticed. I already have a PR in the works that fixes this.
```suggestion
  Scenario: creating a public link with "Editor" role makes it possible to upload a file inside a subdirectory with password set
```